include/openspace/scene/profile.h

```diff
@@ -164,20 +163,19 @@ class Profile {
      * \return The Lua library that contains all Lua functions available for profiles
      */
     static scripting::LuaLibrary luaLibrary();
+
+    std::string file_subName_assets = "";
```

Comment:
This variable is not used anymore and can die

Reply:
Gone